controllers/statemachine/machine_test.go

```diff
@@ -280,3 +281,36 @@ var _ = Context("Issue 271: nextSyncTime not being updated", func() {
 		Expect(m.NextSyncTime().IsZero()).To(BeFalse())
 	})
 })
+
+var _ = Context("Issue 290: Synchronizing condition error doesn't clear", func() {
```

Comment:
The practice of placing an issue# within testing is definitely something that I will have to adopt